modules/appeals_api/app/controllers/appeals_api/supplemental_claims/v0/supplemental_claims/evidence_submissions_controller.rb

```diff
@@ -1,12 +1,64 @@
 # frozen_string_literal: true
 
 module AppealsApi::SupplementalClaims::V0::SupplementalClaims
-  # rubocop:disable Layout/LineLength
-  class EvidenceSubmissionsController < AppealsApi::V2::DecisionReviews::SupplementalClaims::EvidenceSubmissionsController
-    # rubocop:enable Layout/LineLength
+  class EvidenceSubmissionsController < AppealsApi::ApplicationController
+    include AppealsApi::CharacterUtilities
+    include AppealsApi::JsonFormatValidation
     include AppealsApi::OpenidAuth
+    include AppealsApi::Schemas
+    include AppealsApi::StatusSimulation
+    include SentryLogging
+
+    class EvidenceSubmissionRequestValidatorError < StandardError; end
+
+    skip_before_action :authenticate
+    before_action :validate_json_format, if: -> { request.post? }
 
     OAUTH_SCOPES = AppealsApi::SupplementalClaims::V0::SupplementalClaimsController::OAUTH_SCOPES
+    SCHEMA_OPTIONS = { schema_version: 'v0', api_name: 'supplemental_claims' }.freeze
+
+    def show
+      submission = AppealsApi::EvidenceSubmission.find_by(guid: params[:id])
+      raise Common::Exceptions::RecordNotFound, params[:id] unless submission
+
+      submission = with_status_simulation(submission) if status_requested_and_allowed?
+
+      render json: submission,
+             serializer: AppealsApi::EvidenceSubmissionSerializer,
+             key_transform: :camel_lower,
+             render_location: false
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    def create
+      form_schemas.validate!('EVIDENCE_SUBMISSION', params.to_unsafe_h)
```

Comment:
Here I'm using the new `evidence_submission.json` schema I put alongside the existing `200995.json` and `200995_headers.json` schemas - it generates 422 errors as necessary but if anyone has concerns about using `FormSchemas` for this I can revisit. I think if we weren't doing this we'd need to just write some code to manually validate params and throw 422 errors.

Note that `to_unsafe_h` just converts the params object to a hash - I could have used `permit` etc. but this seemed expedient since it's just being validated against the schema

Reply:
That seems all right to me!